### DISCUSSION TITLE
如何修改图片名字的格式？

### DISCUSSION THREAD

**Discussion by lunliang** [TheNetAdmin/zjuthesis#141] [General]
我是用的overleaf模板，模板中的图名的格式好像是黑体加粗，要求好像是和正文一样的格式，应该怎么修改啊
![image](https://user-images.githubusercontent.com/44393167/109627171-4e045500-7b7c-11eb-968c-bbbee99bd80b.png)

**Comment by TheNetAdmin** [comment]
config/format/general/caption.tex

**Reply by lunliang** [reply]
谢谢，能麻烦请教一下怎么修改吗，我LaTeX用的不太熟练，具体不太懂。
![image](https://user-images.githubusercontent.com/44393167/109845228-df142280-7c87-11eb-894c-f1a5aee6b98a.png)
